components/Currency/basic_usage.rst

```diff
@@ -0,0 +1,202 @@
+Basic Usage
+===========
+
+Getting a Currency name
+-----------------------
+
+The ``getName`` method uses Symfony's `Intl`_ class to
+convert currency's code into a human friendly form.
+
+.. _Intl: http://symfony.com/doc/current/components/intl.html
+
+.. code-block:: php
+
+   <?php
+
+   use Sylius\Component\Currency\Model\Currency;
+
+   $currency = new Currency();
+
+   $currency->setCode('USD');
+
+   $currency->getName(); // returns 'US Dollar'
+
+.. note::
+   The output of ``getName`` may vary as the name is generated accordingly to the set locale.
+
+.. _component_currency_context_currency-context:
+
+CurrencyContext
+---------------
+
+The **CurrencyContext** gives you the ability to quickly
+set and retrieve currency names from a given :doc:`/components/Storage/index`.
```

Comment:
Actually, CurrenctContext is responsible for keeping the currently selected/used currency. :)